docs/js_library.md

```diff
@@ -1,11 +1,25 @@
 <!-- Generated with Stardoc: http://skydoc.bazel.build -->
 
-Expose some files with DeclarationInfo, like filegroup but can be a dep of ts_project.
+`js_library` is similar to [`filegroup`](https://docs.bazel.build/versions/main/be/general.html#filegroup); there are no Bazel actions to run.
```

Comment:
Just a question, is there any differences between `js_library` and `filegroup` rule?